templates/pimcore5/files/.platform.app.yaml

```diff
@@ -0,0 +1,100 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+
+
+# The runtime the application uses.
+name: pimcore
+type: php:7.3
+
+
+# The relationships of the application with services or other applications.
+#
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+  database: "pimcore_mysqldb:pimcore_admin"   
+  redis: 'rediscache:redis'
+
+# Add additional PHP extensions.
+build:
+  flavor: none
+
+runtime:
+  extensions:
+    - redis
+    - iconv
+    - dom
+    - simplexml
+    - exif
+    - opcache
+    - imagick
+    - fileinfo
+
+variables:
+  env:
+    PIMCORE_INSTALL_ADMIN_USERNAME: "admin"
+    PIMCORE_INSTALL_ADMIN_PASSWORD: "admin"
+  php:
+      memory_limit: 512M
+
+
+# The 'mounts' describe writable, persistent filesystem mounts in the application. The keys are
+# directory paths, relative to the application root. The values are strings such as
+# 'shared:files/PATH', where PATH is a relative path under the mount's source directory.
+mounts:
+    "/var": "shared:files/var"
+    "/web/var": "shared:files/web-var"
+    "/pimcore": "shared:files/pimcore"
+    "/web/pimcore": "shared:files/web-pimcore"
+
+# The configuration of app when it is exposed to the web.
+web:
+  locations:
+    '/':
+      root: web
+      passthru: "/app.php"
+      allow: true
+      rules:
+        ^/videos/(?<resource>.*)$:
+          passthru: "/var/assets/videos/$resource"
+        ^/img/(?<resource>.*)$:
+          passthru: "/var/assets/img/$resource"
+        ^/static/(?<resource>.*)$:
+          passthru: "/web/static/$resource"
```

Comment:
Is `passthru` really the right property here?  Normally that's for mapping to a PHP file in case of file-not-found, but the links here are not PHP files.  What's the intent here?